pandas/core/window/rolling.py

```diff
@@ -92,6 +93,7 @@ def __init__(
         self.win_freq = None
         self.axis = obj._get_axis_number(axis) if axis is not None else None
         self.validate()
+        self._numba_func_cache: Dict = dict()
```

Comment:
I think I missed this in previous review but similar comment as rest; not a block but subtypes make annotations much more insightful to the reader